Doc/library/cmath.rst

```diff
@@ -229,8 +229,10 @@ Classification functions
    tolerance is ``1e-09``, which assures that the two values are the same
    within about 9 decimal digits.  *rel_tol* must be greater than zero.
 
-   *abs_tol* is the minimum absolute tolerance -- useful for comparisons near
-   zero. *abs_tol* must be at least zero.
+   *abs_tol* is the absolute tolerance; it must be nonnegative.
+   When comparing to ``0.0``, the default tolerance values means that only
```

Comment:
Hmm... I'm not sure that this wording clarifies the behavior.
```suggestion
   When comparing to ``0.0``, the relative tolerance value means
```

Reply:
I would make a similar change in the other file too.

Reply:
@skirpichev Please re-review. Now that I'm reflecting on this, I'm not sure my suggestion about `relative` is correct. Perhaps it should have been `default absolute`. Thanks!

Reply:
I think it should be `default absolute`.

Reply:
> the default tolerance values

The point was that defaults (both for absolute=0.0 and relative tolerances) means that only 0.0 is close to 0.0.  Maybe this: "*abs_tol* is the absolute tolerance; it must be nonnegative.  Only ``0.0` is close to ``0.0`` per default.  Pass an appropriate absolute tolerance to compare with ``0.0``."

Reply:
Thanks @skirpichev and @Xiaokang2022. Let's use @skirpichev's suggested wording. Thanks!

Reply:
@skirpichev @willingc Please review again. Thanks!

Reply:
I am responding in a fresh comment in which I will request changes.  Please do not merge before that.